src/proxy.rs

```diff
@@ -679,15 +680,19 @@ lazy_static! {
 /// Returns:
 ///     System proxies information as a hashmap like
 ///     {"http": Url::parse("http://127.0.0.1:80"), "https": Url::parse("https://127.0.0.1:80")}
-fn get_sys_proxies() -> SystemProxyMap {
+fn get_sys_proxies(
+    #[allow(unused_variables)] registry_values: Option<RegistryProxyValues>,
```

Comment:
This could be `#[cfg_attr(not(target_os), allow(unused_variables))]`.

Reply:
Neat!